--- findlib-1.5.1/src/bytes/Makefile.orig	2014-06-25 12:23:06.000000000 +0200
+++ findlib-1.5.1/src/bytes/Makefile	2014-06-28 20:37:24.619463407 +0200
@@ -1,6 +1,7 @@
 BYTE_FILES=bytes.cmi bytes.cma
 NATIVE_FILES=bytes.cmx bytes.cmxa bytes.a bytes.cmxs
 OCAMLBUILD=ocamlbuild -classic-display -no-links
+PKGDIR=$(shell ocamlfind printconf destdir)
 
 build: all opt
 
@@ -11,7 +12,7 @@
 	$(OCAMLBUILD) $(NATIVE_FILES)
 
 install: all
-	cd _build/ && ocamlfind install bytes ../META $(BYTE_FILES) -optional $(NATIVE_FILES)
+	cd _build/ && ocamlfind install bytes ../META $(BYTE_FILES) -optional $(NATIVE_FILES) -destdir $(prefix)/$(PKGDIR)
 
 uninstall:
 	ocamlfind remove bytes
